include/seqan3/alignment/configuration/align_config_aligned_ends.hpp

```diff
@@ -292,13 +292,13 @@ class end_gaps
     ~end_gaps()                                      noexcept = default;
 
     //!\brief Construction from at least one sequence end-gap specifier.
-    constexpr end_gaps(ends_t && ...args) noexcept
+    constexpr end_gaps(ends_t const ...args) noexcept
```

Comment:
Why do you copy the arguments?

Reply:
because  they are only one byte big and those cheaper then using lvalues.